packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md

```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
 
 ## Elements
 ### Alert
-Alerts are used to communicate status information to the user without blocking other actions through the user interface. There are two styles that can be applied to an alert depending on whether it is intended as an [inline](#inline-alerts) or [toast](#toast-alerts) alert. Regardless of the styling, the basic elements of all alerts are consistent. Since toast alerts should always be dismissable, only a subset of variations should be used.
+Alerts are used to communicate status information to the user without blocking other actions through the user interface. There are two styles that can be applied to an alert depending on whether it is intended as an [inline](#inline-alerts) or [toast](#toast-alerts) alert. Regardless of the styling, the basic elements of all alerts are consistent.
 
-![alert types](./img/alert-types-2.png)
+<img src="./img/alert-types.png" alt="Alert types" width="868">
+
+Toast alerts should always be dismissable, whereas inline alerts can be both dismissable and non-dismissable
 
 1. **Status area:** Use an icon and color to indicate status or severity of the alert.
 2. **Alert title:** Include a short descriptive message. Long message text will wrap. Text color is coded to match alert type.
 3. **Close (optional):** Dismiss the alert. Required for toast alerts and optional for inline alerts.
-4. **Action (optional):** An action can be placed inside of an alert. Only a single action is supported. Note that the alert can include a close button or a single action, but not both.
-5. **Description (optional):** If a longer message is required, it can be included within the Description field.
+4. **Actions (optional):** One or more actions can be placed inside of an alert.
+5. **Description (optional):** If a longer message is required, it can be included within the Description field. This can also include a navigation link as an alternative to an action.
 
 #### Communicating status
-The status area and alert title are coded to communicate the severity of an alert.
+The status icon, background, and alert title are coded to communicate the severity of an alert.
+
+<img src="./img/alert-status.png" alt="alert styles" width="700">
```

Comment:
Convert to self-closing img tag by adding `/` before the closing bracket:
`<img src="./img/alert-status.png" alt="alert styles" width="700" />`